contracts/vault-managers/VaultManagerChainlinkMainAsset.sol

```diff
@@ -285,29 +287,29 @@ contract VaultManagerChainlinkMainAsset is ReentrancyGuard {
     view
     {
         // main collateral value of the position in USD
-        uint mainUsdValue = oracle.assetToUsd(asset, vault.collaterals(asset, user));
+        uint mainUsdValue_q112 = oracle.assetToUsd(asset, vault.collaterals(asset, user));
```

Comment:
Why is it so? `ChainlinkedOracleMainAsset` doesn't use Q112.

Reply:
The newer implementation is used here, see `ChainlinkOracleMainAsset_Mock` contract. The contract in the external repo will be updated after approval of this PR.